pwndbg/commands/context.py

```diff
@@ -204,7 +200,7 @@ def context_backtrace(frame_count=10, with_banner=True):
     result = []
 
     if with_banner:
-        result.append(pwndbg.color.blue(pwndbg.ui.banner("backtrace")))
+        result.append(pwndbg.color.banner(pwndbg.ui.banner("backtrace")))
```

Comment:
Does it make sense for `pwndbg.ui.banner` to automatically invoke the `color` property as well?  It would shorten this, and make it so that all banners are consistent.